src/Compilers/CSharp/Portable/Binder/Binder.CapturedParametersFinder.cs

```diff
@@ -80,7 +80,7 @@ private CapturedParametersFinder(SynthesizedPrimaryConstructor primaryConstructo
                     string name = "<" + parameter.Name + ">PC__BackingField";
```

Comment:
>PC__BackingField [](http://example.com/codeflow?start=59&length=16)

I know this has PROTOTYPE on it but for a follow up: we should define GeneratedNameKind entry (e..g `P` looks free to use) and add a helper to GeneratedNames that produces the name.

I'd also just keep the name as short as possible to avoid unnecessary metadata bloat. Could be just `<name>P`.

Reply:
I will capture these recommendations in comment. Likely in one of the next PRs